.editorconfig

```diff
@@ -0,0 +1,36 @@
+# https://editorconfig.org/
+# specification: https://editorconfig-specification.readthedocs.io/en/latest/
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{c,h,hh,cc,cci}]
+indent_style = space
+indent_size = 4
+
+[Makefile]
+indent_style = tab
+
+[*.am]
+indent_style = tab
```

Comment:
I suggest merging these two blocks. They are essentially about the same kind of files.

I do not insist on this change.

Reply:
They would have to be represented by the same glob, something like
[Makefile{,.am}]
I have some concerns on readability, I'd prefer to keep them separate